hashicorp/boundary-ui
# refactor: 💡 remove route-action usage for users

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12612

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12612)

## Description
remove route-action usage from user routes.


## How to Test


1. start boundary dev instance
2. validate crudl for users works
3. validate add and remove accounts for users works
4. validate no new warnings or errors appear in console

## Checklist:
